falco-exporter/templates/role.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 rules:
 - apiGroups:
-    - extensions
+    - policy
```

Comment:
With this, we need to bump the falco-exporter version too.

Reply:
Thanks, missed that.